pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -54,11 +54,19 @@ type BuildSpec struct {
 	CacheName      string                      `json:"cacheName"`
 	Env            []corev1.EnvVar             `json:"env"`
```

Comment:
Maybe we can add the RunImage field to the Build Object itself. What do yll think about it?

This way we do not need the Rebase.LatestRunImage

Reply:
Yeah..but would we duplicate that information on the build status as well? Would we explicitly pass this run image to the exporter?

Reply:
Hmm, In the end it is like the Builder itself. On each build you tell it with what RunImage/RunImageMirrors you want this build to be created. The status would be more a confirmation

Reply:
Well, kinda. With the builder, the build spec gets updated with the buildpacks that were used for the build and do not explicitly mentioned the SHA of the used builder AFAIK. With the run image it will seemingly be duplicated information.

Reply:
You pass not a fully qualified run image, and then it would not be duplicated no?

Reply:
That is true..but if we aren;t passing the fully qualified run image, isn't that information duplicated within the builder? Unless we do want to move towards "optionally" treating the run image as something one can set independently on the build? As, I think about it, it does bring in the flexibility much like pack does, but could increase complexity for private run images